assets/js/modules/pagespeed-insights/components/notifications/SetupSuccessNotification.js

```diff
@@ -43,10 +45,15 @@ export default function SetupSuccessNotification( { id, Notification } ) {
 	const [ , setNotification ] = useQueryArg( 'notification' );
 	const [ , setSlug ] = useQueryArg( 'slug' );
 
+	const { dismissNotification } = useDispatch( CORE_NOTIFICATIONS );
+
 	const anchorLink = `#${ ANCHOR_ID_SPEED }`;
 	const onJumpLinkClick = ( event ) => {
 		event.preventDefault();
 
+		dismissNotification( id );
+		onDismiss();
```

Comment:
I know it's declared below and this works, but it's harder to follow with the function being declared after it's used. I'm just going to move the declaration of `onDismiss` to be above this function 👍🏻